src/javascript/JContent/ContentRoute/ContentRoute.jsx

```diff
@@ -32,6 +35,38 @@ export const ContentRoute = () => {
         }
     }, [dispatch, mode, viewMode, FLAT, accordionItem]);
 
+    // Captured area information is used to block delete/move/copy/cut actions on areas
+    useEffect(() => {
+        if (path && language) {
```

Comment:
Should we maybe add a check around here to see if a node is renderable on page builder or not, as not all nodes are not always renderable? For example visiting pages/home/area-main/highlights where it is not renderable in page builder.

FYI there is already a check happening around line 50 for this so maybe we just need to move this whole block below it.